source/reference/template-apps.txt

```diff
@@ -92,6 +92,10 @@ Template Apps Available
 
```

Comment:
a couple of things for this page: 

1. you must add it to the reference section TOC for it to show up on the sidebar
2. you must add a redirect from the previous location to this new in the `config/redirects` file